binstar_client/utils/notebook/__init__.py

```diff
@@ -34,3 +39,11 @@ def notebook_url(upload_info):
     else:
         url = "{}://{}/notebooks{}".format(parsed.scheme, parsed.netloc, parsed.path)
     return url
+
+
+def has_environment(nb_file):
+    try:
```

Comment:
if `nbformat` is defined as `None` if the import fails... why not use that in an `if`, 

```python
if nbformat:
    ...
```